config/zfr_cors.global.php.dist

```diff
@@ -43,5 +43,5 @@ return array(
           * the proper response header.
           */
          // 'allowed_credentials' => false,
-    ),
-);
+    ],
+];
```

Comment:
Probably because they're in comments. I can fix if you'd like.

On Aug 16, 2016 5:01 PM, "Daniel Gimenes" <notifications@github.com> wrote:

> In config/zfr_cors.global.php.dist
> <https://github.com/zf-fr/zfr-cors/pull/38#discussion_r75029118>:
>
> > @@ -43,5 +43,5 @@ return array(
> >            * the proper response header.
> >            */
> >           // 'allowed_credentials' => false,
> > -    ),
> > -);
> > +    ],
> > +];
>
> There are still some array() in this file
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zf-fr/zfr-cors/pull/38/files/a9bbdf9c81cfe3c14553208bb60d4558683168fa#r75029118>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABlVypBqBBLqregns4F7bGyomA2gXeDks5qgjMtgaJpZM4JloTx>
> .
>